roxie/ccd/ccdserver.cpp

```diff
@@ -532,10 +516,13 @@ class CRoxieServerActivityFactoryBase : public CActivityFactory, implements IRox
         CriticalBlock b(statsCrit);
         if (started)
             putStatsValue(&node, "_roxieStarted", "sum", started);
-        if (totalCycles)
-            putStatsValue(&node, "totalTime", "sum", (unsigned) (cycle_to_nanosec(totalCycles)/1000));
-        if (localCycles)
-            putStatsValue(&node, "localTime", "sum", (unsigned) (cycle_to_nanosec(localCycles)/1000));
+        //MORE: The following is information is now duplicated
```

Comment:
This provides backward compatibility for anyone who is looking at the stats values.  Is that required, or shall I just delete this code?